internal/pkg/cli/job_init.go

```diff
@@ -158,10 +160,24 @@ func (o *initJobOpts) Validate() error {
 
 // Ask prompts for fields that are required but not passed in.
 func (o *initJobOpts) Ask() error {
-	if err := o.askJobType(); err != nil {
+	if err := o.askJobName(); err != nil {
 		return err
 	}
-	if err := o.askJobName(); err != nil {
+	localMft, err := o.mftReader.ReadWorkloadManifest(o.name)
+	if err == nil {
+		jobType, err := localMft.WorkloadType()
+		if err != nil {
+			return fmt.Errorf(`read "type" field for job %s from local manifest: %w`, o.name, err)
+		}
+		o.wkldType = jobType
+		log.Infof("Local manifest file for job %s is found; Copilot will create from the existing manifest.", o.name)
```

Comment:
Maybe this instead? 💭 
```suggestion
		log.Infof("Manifest file for job %s already exists. Skipping configuration.\n", o.name)
```
with "Copilot will create from the existing manifest" it's not exactly clear what Copilot will create